ang/civiawards/shared/configs/case-type-buttons.config.js

```diff
@@ -0,0 +1,51 @@
+(function (angular, _, url) {
+  var module = angular.module('civiawards');
+  var AWARDS_CATEGORY_NAME = 'awards';
+  var AWARD_CONFIG_URL = 'civicrm/a/#/awards/';
+
+  module.config(function (CaseTypeButtonsProvider) {
+    var awardCategory = getAwardCategory();
+    var awardCaseTypes = getCaseTypesForCategory(awardCategory.value);
+
+    addConfigurationButtonsToCaseTypes(awardCaseTypes);
+
+    /**
+     * Adds configuration buttons to the given case types. The buttons url point
+     * towards the awards configuration form.
+     *
+     * @param {object[]} caseTypes a list of case types objects.
+     */
+    function addConfigurationButtonsToCaseTypes (caseTypes) {
+      _.forEach(caseTypes, function (caseType) {
+        var caseTypeConfigUrl = url(AWARD_CONFIG_URL + caseType.id);
+
+        CaseTypeButtonsProvider.addButtons(caseType.name, [{
+          url: caseTypeConfigUrl
```

Comment:
As mentioned in the previous PR, we also need to supply the icon class from here.